install-nginx.sh

```diff
@@ -20,7 +24,9 @@ then
             ;;
     esac
 else
-    CONFIG_FILE=/etc/nginx/router_config_local.yaml
+    TEMPLATE_FILE_PLUS=/etc/nginx/nginx-plus-router-mesh.conf.j2
+    TEMPLATE_FILE=/etc/nginx/nginx-router.conf.j2
+    CONFIG_FILE=/etc/nginx/router-mesh_config.yaml
```

Comment:
Let's use default values here and reduce the number of variables that are created.

It would look something like:

```
TEMPLATE_FILE_PLUS=/etc/nginx/nginx-plus-router-mesh.conf.j2
TEMPLATE_FILE=/etc/nginx/nginx-router.conf.j2
CONFIG_FILE=/etc/nginx/router-mesh_config.yaml

if [ "$NETWORK" = "fabric" ]
then
... update the variables ...
fi
```